operator/src/secretContractClient.js

```diff
@@ -26,7 +24,6 @@ class SecretContractClient {
             {
                 gas: engOpts.taskGasLimit,
                 gasPrice: engOpts.taskGasPx,
-                from: accounts[this.accountIndex],
```

Comment:
For the reviewer:
the Enigma instance doesn't need this set in the options, because the `web3` instance already sets it automatically using the `defaultAddress`

Reply:
Does this work for you? I remember it not working before,  and it does not seem to work now.
```
Error: No "from" address specified in neither the given options, nor the default options.
    at Object._executeMethod (node_modules/web3-eth/node_modules/web3-eth-contract/src/index.js:838:45)
    at DealManager.createDealAsync (operator/src/dealManager.js:134:97)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

Reply:
note for future reference: fred's comment was misplaced. the error was due to a change elsewhere.